test/Repositories/BooksRepository.spec.ts

```diff
@@ -0,0 +1,103 @@
+import { BookDTO } from 'App/Dto/BookDTO'
+import { Book } from 'App/Models/Book'
+import BooksRepository from 'App/Repositories/BooksRepository'
+import test from 'japa'
+
+test('BooksRepository', () => {
+  let book: BookDTO
+  let sut: BooksRepository
```

Comment:
quando comecei a ler o código sem me atentar as declarações, fiquei perdido com esse `sut`, isso dificulta a leitura, tornando seu códgio menos legível, e demorando mais para outras pessoas ou até mesmo você no futuro entenr o que é  `sut` no meio do código.

```suggestion
  let booksRepository: BooksRepository
```